go/buildhelper/helper.go

```diff
@@ -0,0 +1,57 @@
+package buildhelper
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/obscuronet/obscuro-playground/go/buildhelper/buildconstants"
+)
+
+func Addr1PK() *ecdsa.PrivateKey {
```

Comment:
I think you should store all the various PKs and addreesses in variables (e.g. `var Addr1PK = Addr1PK()`) - clearer than recalling the function every time.

Reply:
I need clean that up 👍 

Should probably have somewhere that loads these PKs from a file or inMemory.

Reply:
Capture a task in https://github.com/obscuronet/obscuro-internal/issues/338